charts/spire/charts/spire-server/values.yaml

```diff
@@ -78,18 +78,27 @@ dataStorage:
   accessMode: ReadWriteOnce
   storageClass: null
 
-logLevel: info
-jwtIssuer: oidc-discovery.example.org
-
 clusterName: example-cluster
-trustDomain: example.org
 
 bundleConfigMap: spire-server
 
-ca_subject:
-  country: NL
-  organization: Example
-  common_name: example.org
+config:
```

Comment:
Yes. Its working because it can toYaml all of .config. if we did all of the stuff at root, .Values, there would be a bunch of junk that leaks through.

Reply:
For clarification, It allows any setting in the server block to be configured. So it solves overriding just the predefined stuff. We could template it all back out, but would loose the ability to set arbitrary settings within the server block.